Makefile

```diff
@@ -61,6 +61,7 @@ _bootstrap-githooks: clean-githooks
 
 .PHONY: bootstrap
 bootstrap:
+	$(MAKE) MAKEFLAGS= _bootstrap-install-pnpm
```

Comment:
Is npx available if we haven't installed node yet?  Seems like it wouldn't be.

Reply:
Good question. I think `make bs` in the sdk repo doesn't actually install node and assumes you've installed it already